src/Writers/HTMLWriter.jl

```diff
@@ -740,6 +746,8 @@ function mdconvert(link::Markdown.Link, parent)
         path = first(s)
         path = endswith(path, ".md") ? Formats.extension(:html, path) : path
         url = (length(s) > 1) ? "$path#$(last(s))" : Compat.String(path)
+        # Replace any backslashes in links, if building the docs on Windows
+        url = replace(url, '\\', '/')
```

Comment:
I'd say that the `link.url` here should already be without backslashes. If that's not the case then there should be a `replace` somewhere earlier.

The only way we should get backslashes here is if the user has them in the URL, in which case it's an user error. We could then either warn, replace them but in the parsing step, or just ignore and assume that that's what the user wants for some reason.